src/index.ts

```diff
@@ -25,8 +26,26 @@ export class WafPlugin {
   }
 
   updateStack() {
+    const generateExcludeRuleList = (rules: string[]) => {
+      return rules.map((ruleToExclude) => ({
+        Name: ruleToExclude,
+      }));
+    };
     // set the waf name based on stage and service name ${self:service}-${self:custom.stage}-webacl
-    const wafName = this.serverless.service.custom.wafName;
+    const wafName = `${
+      this.serverless.service.custom.stage
+    }-${this.serverless.service.getServiceName()}-webacl`;
+
+    const awsCommonExcludeRules: string[] =
+      this.serverless.service.custom?.awsCommonExcludeRules ?? [];
+
+    const awsIpReputationExcludeRules: string[] =
+      this.serverless.service.custom?.awsIpReputationExcludeRules ?? [];
+    this.serverless.service.custom?.ddosRateLimitRules ?? [];
```

Comment:
@13bfrancis did you intend for them to not be able to override the DDOS rule?
I'm thinking for the follow up Jira for logging it would be nice to add the ability to change the DDOS constant of `5000` via a custom parameter as well.

Reply:
That's not a bad idea. I will go ahead and make that change in a separate PR in the future 👍